### PR TITLE
Indexes name are now sorted by name

### PR DESCRIPTION
**Pull request by Cronos87**

### CODE REVIEW

**Comment by MarinPostma** [comment]
I think this is something we want to do directly in the core engine.

**Comment by Cronos87** [comment]
Hello @MarinPostma! Thank you for the reply!

I would like to do it in the core, but sadly I don't know yet Rust, so I prefer avoid doing a mistake in the core 😅

But thank you for the proposition.

File: meilisearch-http/public/interface.html [hunk 1]
```diff
@@ -203,6 +203,10 @@ <h1 class="title is-1 is-spaced">
 
         if (!Array.isArray(result)) { return }
 
+        result.sort(function(a, b) {
+            return a.name.localeCompare(b.name);
```
Comment (fflorent) [inline meilisearch-http/public/interface.html:207]: Beware of performance issues with `String.prototype.localeCompare`. https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/localeCompare#performance

You'd may rather use Intl.Collator: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/Collator

Hope to help :)

**Comment by curquiza** [comment]
Hello @Cronos87!

Thanks for your suggestion of improvement! However, we are currently working hard on a new search engine that would change the way the indexes are managed. That's why we don't accept PR about this for the moment since it will be outdated in few months, but thanks for your involvement in MeiliSearch 🙂

**Pull request closed without merging**